Tests/FoundationTests/FileManagerExtensionsTests.swift

```diff
@@ -80,6 +80,28 @@ final class FileManagerExtensionsTests: XCTestCase {
         } catch {}
     }
 
+    func testEncodePlist() {
+        let point = CGPoint(x: 1.1, y: 2.2)
+        let temporaryDirectoryURL = FileManager.default.temporaryDirectory
```

Comment:
Get the `FileManager` object once and reuse it, instead of constantly referencing `FileManager.default`. E.g.
```Swift
let fileManager = FileManager.default
```